### PR TITLE
[BEAM-5980] Remove redundant combine tests

### PR DESCRIPTION
**Pull request by lgajowy**
Size of input records is irrelevant for the combine operation because the byte[] value gets mapped to a Long before the Combine actually happens. This makes the tests redundant as size is the only thing that changes in the synthetic source options given to the input. 

------------------------

Thank you for your contribution! Follow this checklist to help us incorporate your contribution quickly and easily:

 - [ ] [**Choose reviewer(s)**](https://beam.apache.org/contribute/#make-your-change) and mention them in a comment (`R: @username`).
 - [ ] Format the pull request title like `[BEAM-XXX] Fixes bug in ApproximateQuantiles`, where you replace `BEAM-XXX` with the appropriate JIRA issue, if applicable. This will automatically link the pull request to the issue.
 - [ ] If this contribution is large, please file an Apache [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf).

Post-Commit Tests Status (on master branch)
------------------------------------------------------------------------------------------------

Lang | SDK | Apex | Dataflow | Flink | Gearpump | Samza | Spark
--- | --- | --- | --- | --- | --- | --- | ---
Go | [Build Status](https://builds.apache.org/job/beam_PostCommit_Go/lastCompletedBuild/) | --- | --- | [Build Status](https://builds.apache.org/job/beam_PostCommit_Go_VR_Flink/lastCompletedBuild/) | --- | --- | [Build Status](https://builds.apache.org/job/beam_PostCommit_Go_VR_Spark/lastCompletedBuild/)
Java | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Apex/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Dataflow/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Flink/lastCompletedBuild/)<br>[Build Status](https://builds.apache.org/job/beam_PostCommit_Java_PVR_Flink_Batch/lastCompletedBuild/)<br>[Build Status](https://builds.apache.org/job/beam_PostCommit_Java_PVR_Flink_Streaming/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Gearpump/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Samza/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Spark/lastCompletedBuild/)<br>[Build Status](https://builds.apache.org/job/beam_PostCommit_Java_PVR_Spark_Batch/lastCompletedBuild/)
Python | [Build Status](https://builds.apache.org/job/beam_PostCommit_Python2/lastCompletedBuild/)<br>[Build Status](https://builds.apache.org/job/beam_PostCommit_Python35/lastCompletedBuild/)<br>[Build Status](https://builds.apache.org/job/beam_PostCommit_Python36/lastCompletedBuild/)<br>[Build Status](https://builds.apache.org/job/beam_PostCommit_Python37/lastCompletedBuild/) | --- | [Build Status](https://builds.apache.org/job/beam_PostCommit_Py_VR_Dataflow/lastCompletedBuild/)<br>[Build Status](https://builds.apache.org/job/beam_PostCommit_Py_ValCont/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PreCommit_Python_PVR_Flink_Cron/lastCompletedBuild/) | --- | --- | [Build Status](https://builds.apache.org/job/beam_PostCommit_Python_VR_Spark/lastCompletedBuild/)

Pre-Commit Tests Status (on master branch)
------------------------------------------------------------------------------------------------

--- |Java | Python | Go | Website
--- | --- | --- | --- | ---
Non-portable | [Build Status](https://builds.apache.org/job/beam_PreCommit_Java_Cron/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PreCommit_Python_Cron/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PreCommit_Go_Cron/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PreCommit_Website_Cron/lastCompletedBuild/) 
Portable | --- | [Build Status](https://builds.apache.org/job/beam_PreCommit_Portable_Python_Cron/lastCompletedBuild/) | --- | ---

See [.test-infra/jenkins/README](https://github.com/apache/beam/blob/master/.test-infra/jenkins/README.md) for trigger phrase, status and link of all Jenkins jobs.

### CODE REVIEW

**Comment by lgajowy** [comment]
Run seed job

**Comment by lgajowy** [comment]
@pabloem @kkucharc could you take a look?

**Comment by lgajowy** [comment]
You are right - I will remove there too.

**Comment by lgajowy** [comment]
Run seed job

**Comment by lgajowy** [comment]
It seems that I didn't mess up the jobs. Could you take a look again @kkucharc ?

File: .test-infra/jenkins/job_LoadTests_Combine_Flink_Python.groovy [hunk 1]
```diff
@@ -149,22 +83,15 @@ def batchLoadTestJob = { scope, triggeringContext ->
...
     infra.prepareFlinkJobServer(scope, flinkVersion, dockerRegistryRoot, dockerTag)
-    infra.setupFlinkCluster(scope, jenkinsJobName, flinkDownloadUrl, pythonHarnessImageTag, jobServerImageTag, numberOfWorkers)
 
-    def testConfigs = loadTestConfigurationsSixteenWorkers(datasetName)
-    for (config in testConfigs) {
-        loadTestsBuilder.loadTest(scope, config.title, config.runner, CommonTestProperties.SDK.PYTHON, config.jobProperties, config.itClass)
-    }
-
-    infra.scaleCluster(scope, jenkinsJobName, scaledNumberOfWorkers)
+    def numberOfWorkers = 16
```
Comment (kkucharc) [inline .test-infra/jenkins/job_LoadTests_Combine_Flink_Python.groovy]: What do you think about making it global again and put in parallelism param in each test configuration?

Comment (lgajowy) [reply .test-infra/jenkins/job_LoadTests_Combine_Flink_Python.groovy]: 👍

**Comment by lgajowy** [comment]
Run seed job

**Comment by lgajowy** [comment]
Let's wait unitl monday with this PR. There's ongoing discussion in the proposal whether all of them should be deleted or not

**Comment by pabloem** [comment]
LGTM. Feel free to self-merge if necessary.

**Comment by lgajowy** [comment]
Run seed job

**Comment by lgajowy** [comment]
I left #1 test not deleted as @angoenka suggested. Removing only #2 and #3 test now. See the discussion: https://docs.google.com/a/polidea.com/document/d/1PuIQv4v06eosKKwT76u7S6IP88AnXhTf870Rcj1AHt4/edit?disco=AAAADZUGzAI

**Comment by lgajowy** [comment]
@mwalenia could you take a quick look as well?

File: .test-infra/jenkins/job_LoadTests_Combine_Flink_Python.groovy [hunk 2]
```diff
@@ -32,7 +33,7 @@ String pythonHarnessImageTag = "${dockerRegistryRoot}/python:${dockerTag}"
 String flinkVersion = '1.7'
 String flinkDownloadUrl = 'https://archive.apache.org/dist/flink/flink-1.7.0/flink-1.7.0-bin-hadoop28-scala_2.11.tgz'
 
-def loadTestConfigurationsFiveWorkers = { datasetName -> [
+def scenarios = { datasetName -> [
```
Comment (mwalenia) [inline .test-infra/jenkins/job_LoadTests_Combine_Flink_Python.groovy]: It seems that `datasetName` is not used. Is that intentional?

Comment (lgajowy) [reply .test-infra/jenkins/job_LoadTests_Combine_Flink_Python.groovy]: it is used to fill `metrics_dataset` according to triggering context (pr/not a pr).

**Comment by mwalenia** [comment]
LGTM, thanks, @lgajowy !

**Comment by lgajowy** [comment]
Run seed job

**Comment by lgajowy** [comment]
I squashed some commits...

**Pull request merged**